linkcheck/htmlutil/linkparse.py

```diff
@@ -175,6 +175,10 @@ def start_element (self, tag, attrs, element_text, lineno, column):
         log.debug(LOG_CHECK, "line %d col %d", lineno, column)
         if tag == "base" and not self.base_ref:
             self.base_ref = attrs.get("href", u'')
+        if tag =="a" and attrs.get_true('class', u''):
```

Comment:
This is breaking our unit tests because

> E   AttributeError: 'dict' object has no attribute 'get_true'

(also there's a missing space after the `==`)